inc/classes/class-plugin.php

```diff
@@ -154,13 +154,16 @@ function() {
 			'plugin_updater_subscriber',
 			'capabilities_subscriber',
 			'varnish_subscriber',
-			'cloudflare_subscriber',
 			'rocketcdn_rest_subscriber',
 			'detect_missing_tags_subscriber',
 			'purge_actions_subscriber',
 		];
 
-		if ( \rocket_valid_key() ) {
+		if ( get_rocket_option( 'do_cloudflare' ) ) {
```

Comment:
same here

Reply:
See my notes above. It's actually a little faster with this function.